src/main/cpp/CMakeLists.txt

```diff
@@ -23,6 +23,12 @@ include(rapids-find)
 include(rapids-cpm)
 rapids_cpm_init()
 
+# find libcu++
+include(${rapids-cmake-dir}/cpm/libcudacxx.cmake)
+
+# find thrust/cub
+include(../../../thirdparty/cudf/cpp/cmake/thirdparty/get_thrust.cmake)
+
```

Comment:
Should this be before the `project` declaration?  If moved lower, we could leverage the `CUDF_DIR` variable which others are using to find files within thirdparty/cudf.

Reply:
Thanks. I've moved that down.